t/lib/t/MusicBrainz/Server/Edit/Release/Edit.pm

```diff
@@ -175,6 +175,36 @@ test 'A missing comment does not clear an existing one' => sub {
     is($release->comment, 'hello', 'comment is left unchanged');
 };
 
+test 'MBS-13300: Release group cover art is deleted if the release is moved' => sub {
```

Comment:
```suggestion
test 'MBS-13300: Release group cover art is unset if the release is moved' => sub {
```
more clearly shows that we don't expect the cover art itself to be deleted, just not set for the RG.